### PR TITLE
Revamp admin usage docs

### PR DESCRIPTION
**Pull request by askvortsov1**
- Reorganize some docs pages to have a clearer order
- Remove unnecessary pages (appearance, permissions, tags): these are self-evident from the admin dashboard
- (Slightly) document extensions
- (Slightly) document theming options

### CODE REVIEW

File: docs/admin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Admin Dashboard
+
+The Flarum Admin Dashboard is a user-friendly interface for managing your forum.
+It is only available to users in the "admin" [group](permissions.md).
+To access the admin dashboard, Click on your **Name** at the at the top right of the screen, and select **Administration**.
```
Comment (clarkwinkelmann) [inline docs/admin.md:5]: Name => Username?

File: docs/admin.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Admin Dashboard
+
+The Flarum Admin Dashboard is a user-friendly interface for managing your forum.
+It is only available to users in the "admin" [group](permissions.md).
```
Comment (clarkwinkelmann) [inline docs/admin.md:4]: Capitalization of admin? => Admin?

File: docs/extensions.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+Flarum is minimalistic, but it's also highly extensible. In fact, most of the features that ship with Flarum are actually extensions!
+
+This approach makes Flarum extremely customizable: You can disable any features you don't need, and install other extensions to make your forum perfect for your community.
+
+For more information on Flarum's philosophy on what features we include in core, or if you're looking to make your own extension, please see our [extension documentation](extend/README.md).
+This article will focus on managing extensions from a forum admin's perspective.
+
+## Finding Extensions
+
+Flarum has a wide ecosystem of extensions, most of which are open source and free. To find new and awesome extensions, go to the [Extensions](https://discuss.flarum.org/t/extensions) tag on Flarum's community forums. The unofficial [Extiverse extension database](https://extiverse.com/) is also a great resource.
```
Comment (clarkwinkelmann) [inline docs/extensions.md:12]: go => visit?

File: docs/extensions.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+
+## Installing Extensions
+
+Just like Flarum, extensions are installed through [Composer](https://getcomposer.org), using a terminal. To install a typical extension:
```
Comment (clarkwinkelmann) [inline docs/extensions.md:16]: a terminal => the terminal?

File: docs/extensions.md [hunk 5]
```diff
@@ -1 +1,27 @@
...
+
+1. `cd` to the folder containing `composer.json`.
+2. Run `composer require COMPOSER_PACKAGE_ID`. This should be provided by the extension's documentation.
```
Comment (clarkwinkelmann) [inline docs/extensions.md:19]:
```suggestion
2. Run `composer require COMPOSER_PACKAGE_NAME`. This should be provided by the extension's documentation.
```

**Review by clarkwinkelmann — COMMENTED** [review]
Some wording suggestions

**Review by clarkwinkelmann — APPROVED** [review]
Regarding the terminal vs SSH nomenclature, we should probably make sure we're consistent across our documentation.

While I think terminal/shell represents the concept independently from the method of access, SSH is a specific protocol that allows accessing a terminal/shell over the network. But the terminal can also be accessed over other tools like telnet (unlikely because of security), or a web terminal, for example on DigitalOcean.

**Comment by askvortsov1** [comment]
I changed it to SSH to be more consistent with the install docs

**Review by tankerkiller125 — APPROVED** [review]

**Pull request merged**